### ISSUE TITLE
Error when removing graph

### ISSUE THREAD

**Issue by bergie** [flowhub/the-graph#329]
When you set the graph of `the-graph-editor` to `NULL` (for example, when navigating away from a graph in noflo-ui), we get an error:

![screenshot 2017-02-23 at 12 23 43](https://cloud.githubusercontent.com/assets/3346/23257196/10044844-f9c3-11e6-93c9-f93e00ef2d15.png)

**Comment by jonnor** [comment]
I suspect there are other things broken in this case also, like stale state in component library etc. As far as I know, noflo-ui used to just swap the entire widget, right?

**Comment by bergie** [comment]
@jonnor nope, we keep one widget and change graph. Current `master` of noflo-ui keeps the last-seen graph in state until a new one is provided, which causes occasional funkiness. This is fixed in noflo/noflo-ui#680, but that unfortunately causes this issue with `the-graph-editor`

**Comment by jonnor** [comment]
I'm tempted to consider `graph` property mandatory, and close this as wont-fix - unless there is an important usecase for it.
